notebooks/01_thicket_tutorial.ipynb

```diff
@@ -181,8 +181,55 @@
     "tags": []
    },
    "source": [
-    "## 4. Thicket Components\n",
+    "## 4. Thicket Components"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3fe4f805",
+   "metadata": {},
+   "source": [
+    "### 4.i Profiles\n",
```

Comment:
What is section 4.i? I see that you're using 4.1, 4.2, .. going forward.

Reply:
I was trying to distinguish the profiles section from the other sections, since we historically haven't had users interface with the profile objects. But they could just be `4.1 Profiles`, `4.2 Performance Data`, ...